lib/lite_rolling_switch.dart

```diff
@@ -91,20 +90,20 @@ class _RollingSwitchState extends State<LiteRollingSwitch>
 
   @override
   Widget build(BuildContext context) {
-    Color transitionColor = Color.lerp(widget.colorOff, widget.colorOn, value);
+    Color? transitionColor = Color.lerp(widget.colorOff, widget.colorOn, value);
 
     return GestureDetector(
       onDoubleTap: () {
         _action();
-        if (widget.onDoubleTap != null) widget.onDoubleTap();
+        widget.onDoubleTap();
```

Comment:
Null check before function invocation would be a better approach (just as example)

```dart
widget.onDoubleTap!();
```

Reply:
All events except onChange will remain as optional, null check on function invocation is used for newer flutter version compatibility.